src/tests/functional/shared_test_classes/src/single_op/stft.cpp

```diff
@@ -55,6 +55,10 @@ void STFTLayerTest::SetUp() {
     ElementType step_size_type;  // size/step type
     utils::InputLayerType param_type;
 
+    // With usage of RDFT executor, in favour of performance,
+    // the results are slightly different than reference impl
+    abs_threshold = 1e-1;
```

Comment:
I see that for rdft test we also have to increase the threshold, but in a much lesser extent (https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/src/single_op/rdft.cpp#L87-L89). 
1e-1 usually too high for fp32 math. Do you have any ideas why it needs to be so high?

Reply:
I will investigate that deeper, but first thing that comes to my mind is different test input data. 
Also it's not like every result value needs abs_threshold = 0.1, but there are few.

Reply:
Update in: https://github.com/openvinotoolkit/openvino/pull/26967#issuecomment-2548123391